_includes/1.0/overview/security/oauth2/authorization.html

```diff
@@ -0,0 +1,81 @@
+                <th>Description</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>response_type</td>
+                <td><b>code *</b></td>
+                <td>Indicates that the client expects to receive an authentication code that will later be exchanged by an access token</td>
+            </tr>
+            <tr>
+                <td>client_id</td>
+                <td><i>your_client_id</i></td>
+                <td>The unique id that was provided to you after your client credential was created</td>
+            </tr>
+            <tr>
+                <td>redirect_uri</td>
+                <td><i>your_redirect_uri</i></td>
+                <td>The callback url that was provided to us when you requested your client credential</td>
+            </tr>
+            <tr>
+                <td>scope</td>
+                <td><b>offline *</b></td>
+                <td>The only scope available. It indicates that you will have read access to user's data and you will receive a <i>refresh token</i> when exchanging the authentication code</td>
+            </tr>
+            <tr>
+                <td>state</td>
+                <td><i>any_string</i></td>
+                <td>This value must be present in the authorization response and must be the same as the one you sent in the request. <br>For more information on how to take advantage of this parameter, check <a href="https://auth0.com/docs/protocols/state-parameters" target="_blank">this article</a></td>
+            </tr>
+        </tbody>
+    </table>
+    <p style="text-align: right"><small><b>* only one possible valid value</b></small></p>
+
+{% highlight http %}
+GET {{site.oauth2.endpoints.authorization.url}}?response_type=code&client_id=xxxxxxxxxx&redirect_uri=xxxxxxxxxx&scope=offline&state=xxxxxxxxxx HTTP/1.1
+Host: {{ site.oauth2.domain }}
+{% endhighlight %}
+
+{% highlight bash %}
+curl -D- --request GET '{{site.oauth2.protocol}}://{{ site.oauth2.domain }}{{site.oauth2.endpoints.authorization.url}}'\
+'?response_type=code'\
+'&client_id=xxxxxxxxxx'\
+'&redirect_uri=xxxxxxxxxx'\
+'&scope=offline_access'\
+'&state=xxxxxxxxxx'
+{% endhighlight %}
+
+    <p>The OneUp Authorization Server response will be a redirect to the login page:</p>
+
+    <img alt="login page" src="/img/login_page.png" style="height: 100%; width: 100%; object-fit: contain; margin-bottom: 10px"/>
+
+    <p>The user fills the login information and press the <kbd>sign in</kbd> button.</p>
+
+    <p>The authentication server authenticates the user and redirects him back to the Authorization Server.</p>
+
+    <p>The OneUp Authorization Server redirects the user to the consent page:</p>
+
+    <img alt="consent page" src="/img/consent_page.png" style="height: 100%; width: 100%; object-fit: contain; margin-bottom: 10px"/>
+
+    <p>The user press the <kbd>allow</kbd> button. The web application redirects the user to the Authorization Server.</p>
```

Comment:
Can you add a comment on what happens if the user press "Deny"? Is he redirected to the callback URL with an error code ?

Reply:
Added with the commit [908f6b3724d13700528479bf0e305807c8ca9161](https://github.com/myERP/myerp.github.io/pull/31/commits/908f6b3724d13700528479bf0e305807c8ca9161)